python/pyfeelpp-toolboxes/feelpp/toolboxes/fluid/fluid.cpp

```diff
@@ -88,7 +88,12 @@ void defFM(py::module &m)
         .def("exportResults",static_cast<void (fm_t::*)( double )>(&fm_t::exportResults), "export the results of the fluid mechanics problem", py::arg("time"))
 
         // remesh
-        .def("applyRemesh",&fm_t::applyRemesh, "apply remesh to toolbox and regenerate the necessary data structure")
+        .def("applyRemesh",
```

Comment:
@lberti please add `py::arg` for each parameters

Reply:
Done